doc/source/myrocks/variables.rst

```diff
@@ -951,6 +959,19 @@ Specifies whether MyRocks should create new column families
 if they do not exist.
 Disabled by default.
 
+.. variable:: rocksdb_create_temporary_checkpoint
+
+  :cli: ``--rocksdb-create-temporary-checkpoint``
+  :dyn: Yes
+  :scope: Session
+  :vartype: String
+
+This variable has been implemented in |Percona Server| :rn:`8.0.15-6`.
```

Comment:
George should review this, but IMHO "this is to be used by backup tools" or similar has to added, just like the other variables has it

Reply:
Agree with @laurynas-biveinis we need a warning similar to something like "These variables are only to be used by automated backup tools.  Prolonged use or other misuse can have serious side effects to the server instance." or some other more proper English ;)